src/styles/variables.less

```diff
@@ -1,4 +1,4 @@
-@card-icon-lg:                                        108px;
+@card-icon-lg:                                        72px;
```

Comment:
Yeah, I left it in case we ever wanted to change it to be different but I can remove it.